File: VB/WinSolution.Module/DomainObject3.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Persistent.Base
Imports DevExpress.Xpo

Namespace WinSolution.Module
	<DefaultClassOptions> _
	Public Class DomainObject3
		Inherits DomainObjectBase
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private _propertyName3 As String
		Public Property PropertyName3() As String
			Get
				Return _propertyName3
			End Get
			Set(ByVal value As String)
				SetPropertyValue("PropertyName3", _propertyName3, value)
			End Set
		End Property
	End Class
End Namespace

File: CS/WinSolution.Module/DomainObject3.cs
using DevExpress.Persistent.Base;
using DevExpress.Xpo;

namespace WinSolution.Module {
    [DefaultClassOptions]
    public class DomainObject3 : DomainObjectBase {
        public DomainObject3(Session session) : base(session) { }
        private string _propertyName3;
        public string PropertyName3 {
            get { return _propertyName3; }
            set { SetPropertyValue("PropertyName3", ref _propertyName3, value); }
        }
    }
}
